uui/components/pickers/PickerInput.fn.tsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+import { PickerBodyBaseProps, PickerInputBaseProps, PickerTogglerProps, usePickerInput } from '@epam/uui-components';
+import { Dropdown } from '../overlays/Dropdown';
+import { PickerInputProps } from './PickerInput';
+import { EditMode, IHasEditMode, SizeMod } from '../types';
+import { DataRowProps, DataSourceListProps, DropdownBodyProps, IDropdownToggler, IEditableDebouncer, isMobile, uuiMarkers } from '@epam/uui-core';
+import { PickerModal } from './PickerModal';
+import { PickerToggler, PickerTogglerMods } from './PickerToggler';
+import { Panel } from '../layout';
+import { MobileDropdownWrapper } from './MobileDropdownWrapper';
+import { DataPickerBody } from './DataPickerBody';
+import { DataPickerRow } from './DataPickerRow';
+import { DataPickerFooter } from './DataPickerFooter';
+import { PickerItem } from './PickerItem';
+
+import css from './PickerInput.module.scss';
+
+const pickerHeight = 300;
+const pickerWidth = 360;
+
+function PickerInput<TItem, TId>(props: PickerInputBaseProps<TItem, TId> & SizeMod & IHasEditMode) {
+    const toggleModalOpening = () => {
```

Comment:
@jakobz, @AlekseyManetov, this function is passed to the `usePickerInput` hook, and functions from the output of `usePickerInput`  are used inside this function. Pretty weird, but the only option I found.